// [python quickstart](https://grpc.io/docs/quickstart/python.html#run-a-grpc-application)
// python -m grpc_tools.protoc --python_out=. --grpc_python_out=. -I../../proto common.proto


syntax = "proto3";

service Common {
    rpc common_infer(CommonRequest) returns (CommonReply) {}
}

message CommonRequest {
    bytes input_bytes = 1;
}

message CommonReply {
    bytes result_bytes = 1;
}


